README.md

```diff
@@ -70,14 +70,20 @@ Since this list strives for completeness rather than listing only the best, it's
+- [Boarding Missions](https://github.com/tmbutterworth/boarding-missions) - Small plugin that adds chance for being rewarded for assisting disabled Milita, Free worlds, and Navy ships.
 - [Ciu Kreo](https://github.com/RestingImmortal/Ciu-Kreo) - A feature-rich all-content plugin suitable for both content creators and casual players.
 - [Czartrak's Ship Pack](https://github.com/czartrak/Czartraks-Ship-Pack) - Adds several new, mostly human ships.
 - [Endless Sky high-DPI](https://github.com/endless-sky/endless-sky-high-dpi) - The official plug-in for high-DPI monitors.
 - [ES-HUDs](https://github.com/comnom/es-huds) - A collection of HUD plug-ins.
 - [es-plugin-maintenance](https://github.com/comnom/es-plugin-maintenance) - An attempt to keep abandoned plug-ins uptodate.
+- [Flamethrower Turret](https://github.com/ItsNickBarry/endless-sky-flamethrower-turret) - Adds a turret variant of the flamethrower.
 - [Gatling Ammo Fix](https://github.com/Cat-Lady/Gatling-Ammo-Fix/releases) -  Fixes availability of Gatling Gun Ammo (and its boxes) - being most primitive and easiest to manufacture type of ordnance, it is now available where you would expect it to be ("galactic south" and multipurpose ammo depots).
+- [Korath Automation Upgrade](https://github.com/Kriegstofu/Korath-Automation-Upgrade) - A plugin for Endless Sky which adds a new Korath Warship after the end of the Korath/Wanderer Co-Op Mission.
+- [Quarg Military Retaliation](https://groups.google.com/forum/#!msg/endless-sky/IkoSYsZgMlQ/qSZUvTjFAQAJ) - This plugin contains new ships, weapons, outfits, descriptions, and missions for the Quarg. The key feature is Quarg responses to being farmed. The response will start with increased patrols and escalate as farming continues to include hunter-killer groups, defense fleets, carriers, battleships, and more.
 - [swizzle-player](https://github.com/Cat-Lady/swizzle-player/releases) - Allows to change player's swizzle in "Endless Sky" - and keep the colour after ES get updates that change goverments.txt data file.
 - [The Adamas Project](https://github.com/toilethinges/Adamas-Project) - Several new ships, many outfits, and a variety of other things to be discovered.
+- [The Enclave](https://github.com/Makuta-Miras/The-Enclave) - The Enclave focuses around the discovery of an isolated group of humans in a distant galaxy. Separated from the rest of their species for over half a millennium, how have these people developed?
+- [Ursa Polaris](https://steamcommunity.com/sharedfiles/filedetails/?id=631160921) - This plugin contains missions for your pilot to create a centrally located outfitter space station that stocks outfits from across the galaxy. It happens after main campaing and is aiming to be NOT overpowered.
```

Comment:
Is it worth linking the original when it's part of [es-plugin-maintenance](https://github.com/comnom/es-plugin-maintenance)?

Reply:
Yes, cause the es-plugin-maintentance is - quite counter-intuitively to the name - currently unmaintained, and (for example) Ursa Polaris have much more recent version mainstream.

If anything, I would kick out the es-plugin-maintentance from list, or at least add a warning that it is not guaranteed to actually have recent versions of the plugins.

Reply:
cc @comnom

Reply:
Probably in your best interest to take it off. It was made as a service to users, and doesnt account for authors returning. Ive also seen a new version of animated stars forked from the original broken one, so the version people are likely to use doesnt include my fix anyhow. A better solution would be for me to take stuff like that out, but unless someone else wants to take over that repo (ha), that aint gonna happen.